sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -949,6 +949,29 @@ class BlobAPITest extends APISpec {
         bc.getPropertiesWithResponse(null, null, null).getStatusCode() == 200
     }
 
+    @Unroll
+    def "Get properties rehydrate priority"() {
+        setup:
```

Comment:
Putting this test here feels a little disjointed to me. It seems like we should add this property to the list in the test above and then also add a getProperties call to the tests for setTier that test rehydratePriority (which I'm not actually finding any that test that?)

Reply:
Ok, I can make this a setTier test and just check that the rehydrate priority is null on a regular get properties call

Reply:
Modified the test